core/lib/libtomcrypt/src/misc/mem_neq.c

```diff
@@ -74,6 +74,11 @@ int mem_neq(const void *a, const void *b, size_t len)
       ++pb;
    }
 
+   ret |= ret >> 4;
+   ret |= ret >> 2;
+   ret |= ret >> 1;
+   ret &= 1;
+
```

Comment:
While being safe and correct is a lot more complicated than `!!ret`, which otherwise often is used to make sure either `true` or `false` is returned.